### ISSUE TITLE
This check was skipped view should hint how to enable debugging

### ISSUE THREAD

**Issue by jsoref** [actions/runner#1131]
If you have a feature request that is relevant to this repository, the runner, then please include the information below:

**Describe the enhancement**

Currently, it's possible to get a view like:
https://github.community/t/jobs-being-skipped-while-using-both-needs-and-if-unclear-why/18480/4
![image](https://user-images.githubusercontent.com/2119212/120670118-1ac4b100-c45e-11eb-9af6-5ee82e1c688e.png)

It'd be nice if that view had a little hint in the corner:

###### [ℹ️ Enable step logging to see why in future runs](https://docs.github.com/en/actions/managing-workflow-runs/enabling-debug-logging#enabling-step-debug-logging)

In theory that should result in something like: https://github.community/t/when-an-action-is-skipped-why-doesnt-the-log-detail-why/179322/2

**Code Snippet**
If applicable, add a code snippet.

**Additional information**
Add any other context about the feature here.

NOTE: if the feature request has been agreed upon then the assignee will create an ADR.   See docs/adrs/README.md

**Comment by ethomson** [comment]
I agree - it would be nice to have more information about this.  I'm adding this to the backlog.
